repositories.yaml

```diff
@@ -1083,3 +1083,25 @@ applications:
       - v1_name: rally-attention-stream
         app_id: rally-attention-stream
     skip_documentation: true
+
+- app_name: viu_politica
```

Comment:
I believe this is missing some fields that'll allow this to properly run (v1_name and app_id, specifically). Did you happen to test this? Maybe with a command like `python -m probe_scraper.runner --glean --glean-repo viu_politica`?